frontend/src/metabase/dashboard/containers/CreateDashboardModal.unit.spec.tsx

```diff
@@ -73,15 +73,18 @@ describe("CreateDashboardModal", () => {
 
   it("can't submit if name is empty", async () => {
     setup();
-    const submitButton = await waitFor(() =>
-      screen.getByRole("button", { name: "Create" }),
-    );
-    expect(submitButton).toBeDisabled();
+    expect(
+      await screen.findByRole("button", { name: "Create" }),
+    ).toBeDisabled();
   });
 
-  it("calls onClose when Cancel button is clicked", () => {
+  it("calls onClose when Cancel button is clicked", async () => {
```

Comment:
This is strange. all `userEvent` events should be wrapped in act automatically

Reply:
Yeah, not sure what's going on there, but I'm getting errors about calls not being wrapped in `act` :/

Reply:
fwiw, just awaiting the `userEvent.click` call doesn't help

Reply:
Fixed by replacing `act` with `waitFor` wrapping the assertions. Caused by async validation in formik https://github.com/jaredpalmer/formik/issues/1543